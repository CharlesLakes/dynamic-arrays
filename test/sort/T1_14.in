10
76 22 70 84 69 87 90 23 48 82