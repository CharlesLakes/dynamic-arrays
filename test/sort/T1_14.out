22 23 48 69 70 76 82 84 87 90